### DISCUSSION TITLE
Why are so many features outside the GitHub mobile app?

### DISCUSSION THREAD

**Discussion by TechStudent10** [community/community#39004] [Mobile]
Title. As an example, I got a security notification about one of my repos in the GitHub mobile app for Android. When I tapped on it, I was prompted to open it in the web browser. Why?

Another example was when I tried creating an issue frok an issue form. These have been around for months, yet they aren't integrated into the mobile app. 

Is there something I'm missing here?

**Comment by dhrubasaha08** [comment]
No, bro thats how github mobile app work. It pretty trash compare to its website!

**Comment by TechStudent10** [comment]
Hello GitHub?
